### PR TITLE
refactor: Structural output Android models describe

### LINKED ISSUE #1846: Structural output - DescribeAndroidModels
# Plan

According to the specification described in epic #1728, implement the structural output for the command and move the console logging to layer CLI.

# Command

`DescribeAndroidModels`

### PR DESCRIPTION
**Pull request by piotradamczyk5**
Fixes #1846

- Code is refactored according to #1846
- Android describe models command works like previously

## Checklist

- [x] Unit tested

### CODE REVIEW

File: test_runner/src/main/kotlin/ftl/presentation/cli/firebase/test/android/models/AndroidModelDescribeCommand.kt [hunk 1]
```diff
@@ -45,4 +49,12 @@ class AndroidModelDescribeCommand :
     override var modelId: String = ""
 
     override fun run() = invoke()
+
+    override val out: Any.() -> Unit
```
Comment (pawelpasterz) [inline test_runner/src/main/kotlin/ftl/presentation/cli/firebase/test/android/models/AndroidModelDescribeCommand.kt:53]:
```suggestion
    override val out = outputLogger { 
```

**Review by pawelpasterz — APPROVED** [review]
Minor suggestion, LGTM!

**Review by Sloox — CHANGES_REQUESTED** [review]
small required change from @pawelpasterz

**Comment by piotradamczyk5** [comment]
> small required change from @pawelpasterz

@Sloox fixed

**Review by Sloox — APPROVED** [review]

**Pull request merged**